src/components/empty_prompt/empty_prompt.stories.tsx

```diff
@@ -0,0 +1,101 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+import React from 'react';
+import type { Meta, StoryObj } from '@storybook/react';
+
+import { EuiEmptyPrompt, EuiEmptyPromptProps } from './empty_prompt';
+
+import { EuiButton, EuiButtonEmpty } from '../button';
+import { EuiTitle } from '../title';
+import { EuiLink } from '../link';
+import { EuiImage } from '../image';
+import { EuiPageTemplate } from '../page_template';
+import illustration from '../../../src-docs/src/images/empty-prompt/illustration.svg';
```

Comment:
[optional import organization nit] in general, as a pattern across EUI, we should prefer to order from most generic to most specific. So `EuiEmptyPrompt` should come last, and generic components before:

```suggestion
import React from 'react';
import type { Meta, StoryObj } from '@storybook/react';

import illustration from '../../../src-docs/src/images/empty-prompt/illustration.svg';
import { EuiButton, EuiButtonEmpty } from '../button';
import { EuiTitle } from '../title';
import { EuiLink } from '../link';
import { EuiImage } from '../image';
import { EuiPageTemplate } from '../page_template';

import { EuiEmptyPrompt, EuiEmptyPromptProps } from './empty_prompt';
```